src/auditlog/models.py

```diff
@@ -94,6 +94,9 @@ def get_for_objects(self, queryset):
 
         if isinstance(primary_keys[0], integer_types):
             return self.filter(content_type=content_type).filter(Q(object_id__in=primary_keys)).distinct()
+        elif isinstance(queryset.model._meta.pk, models.UUIDField):
```

Comment:
It provides a `uuid.UUID` instance, which errors when compared with the string in the `object_pk` field. Therefore I have to convert the UUIDs to strings.

I would use the new [`Cast` function](https://docs.djangoproject.com/en/1.10/ref/models/database-functions/#cast), but it was added in Django 1.10, so the user of django-auditlog would have to use Django 1.10 for this to work.